pkg/controller/volume/persistentvolume/pv_controller.go

```diff
@@ -1250,7 +1250,7 @@ func (ctrl *PersistentVolumeController) provisionClaimOperation(claimObj interfa
 		// This means that an unknown provisioner is requested. Report an event
 		// and wait for the external provisioner
 		if storageClass != nil {
-			msg := fmt.Sprintf("cannot find provisioner %q, expecting that a volume for the claim is provisioned either manually or via external software", storageClass.Provisioner)
+			msg := fmt.Sprintf("waiting for external provisioner %q to provision a volume or creating a volume mannually for the claim", storageClass.Provisioner)
```

Comment:
* typo `mannually` -> `manually`
* in my (imperfect) English the sentence "waiting for external provisioner %q to provision a volume or creating a volume manually" means that Kubernetes is either "waiting for external provisioner" or "Kubernetes is creating a volume manually", which is not the case. Kubernetes is waiting for someone to create the volume, be it external provisioner or system admin.